Sources/Apollo/InMemoryNormalizedCache.swift

```diff
@@ -21,6 +21,10 @@ public final class InMemoryNormalizedCache: NormalizedCache {
     return records.merge(records: newRecords)
   }
 
+  public func removeRecords(matching pattern: CacheKey) throws {
+    return records.removeRecords(matching: pattern)
```

Comment:
Do we need the `return` here?

Reply:
No, it's more out of habit - is there a specific style guide we adhere to?